README.md

```diff
@@ -925,15 +926,34 @@ The library provides an `Elao\Enum\Bridge\ApiPlatform\Core\JsonSchema\Type\ElaoE
 
 ## EasyAdmin
 
+### EasyAdmin2
+
+In the form/fields section, set the type to the FQCN of the EnumType and configure it appropriately with type_options.
+
+```yaml
+form: 
+  fields:
+    - { property: language, type: 'Elao\Enum\Bridge\Symfony\Form\Type\EnumType', type_options: { enum_class: 'App\Entity\Enum\LanguageEnum', required: true, }, label: 'Language', help: 'Select language from the drop down above'}
+```
+
+To ensure the listing is working, set the type to text.
+
+```yml
+list:
+  fields:
+    - { property: language, label: 'Language', type: 'text'}
```

Comment:
For this to work, the enum must be a readable enum. Also, it won't translate in case you use translation keys as readable values, nor null values I guess.

A solution for this is to create a custom template `templates/easyadmin/enum.html.twig`, like : 

```twig
{% if value is empty %}
    {{ include(entity_config.templates.label_null) }}
{% else %}
    {{ value|trans }}
{% endif %}
```

and use it like this:

```yml

```yml
list:
  fields:
    - { property: language, label: 'Language', template: easyadmin/enum.html.twig }
```

Reply:
Thank you. I have included your comments in the patch. Hope that was your intention?

Reply:
That's fine, thank you :)